test/integration/client/testdata/exec-plugin.sh

```diff
@@ -18,4 +18,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+if [[ -n "${EXEC_PLUGIN_OUTPUT_FILE-""}" ]]; then
```

Comment:
I really wanted to collapse this shell script into a single line: `cat "$EXEC_PLUGIN_OUTPUT_FILE"`. I got a little bit lazy because it would take an hour or two of refactoring the rest of the test file. I was also cognizant of trying to keep the blast radius smaller on this PR. I can certainly do the refactor, though, if you think that would be valuable.